src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -164,4 +164,12 @@ public void testEmptyStatement() throws Exception {
         testByConfig("EmptyStatement/patchedline/defaultContextConfig.xml");
         testByConfig("EmptyStatement/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/142")
```

Comment:
please explain in the issue what problem we have

Reply:
https://github.com/checkstyle/patch-filters/pull/145#discussion_r460869686

Reply:
>  in the issue
not in pull request comments. Because you're referencing issue 142 in your code.

Reply:
ok, I get it.

Reply:
you updated pull request's description, not issue's description (https://github.com/checkstyle/patch-filters/issues/142)

Reply:
sorry, done.